.changeset/thirty-moose-yawn.md

```diff
@@ -0,0 +1,7 @@
+---
+"@trueplan/forecast-components": minor
+"@trueplan/forecast-theme": minor
+---
+
+[theme]: new line height values. lineHeight[2] key was removed and value moved to lineHeight[5].
+[avatar]: lineHeight prop updated to use new key
```

Comment:
Would you mind trying something for this one. I want to see what separate changeset files does to our change log. So edit this one to be only for components, and then add a new changeset file for theme.

Reply:
done